### ISSUE TITLE
dart:mirrors error import

### ISSUE THREAD

**Issue by AIBuilderExpert** [exitlive/stripe-dart#36]
Im getting an error when i import stripe package.

import "package:stripe/stripe.dart";

Im getting the error while running the project with the next message

Launching lib/main.dart on Gerardo's iPhone in debug mode...
Automatically signing iOS for device deployment using specified development team in Xcode project:
*********
Starting Xcode build...
Xcode build done.                                           31.8s
Installing and launching...
[VERBOSE-2:shell.cc(184)] Dart Error: error: import of dart:mirrors with --enable-mirrors=false
[VERBOSE-2:engine.cc(172)] Could not prepare to run the isolate.
[VERBOSE-2:engine.cc(119)] Engine not prepare and launch isolate.
[VERBOSE-2:FlutterEngine.mm(261)] Could not launch engine with configuration.
Syncing files to device Gerardo's iPhone...

And i get a white screen on my phone

**Comment by enyo** [comment]
Yup that’s because we used mirrors. (It was intended as a server package).

I’ll remove the mirror dependency today.

**Comment by enyo** [comment]
Mh, I just realised that due to the dart `2.x` upgrade the change is more complicated than I anticipated.  @skatzbizarro  If you tell me which resources you are most interested in, I can get that off the ground first.
